includes/utils.inc

```diff
@@ -23,19 +23,26 @@ use Drupal\webform_civicrm\Utils;
  * @return string or integer
  */
 function wf_crm_state_abbr($input, $ret = 'abbreviation', $country_id = NULL) {
-  $input_type = $ret == 'id' ? 'abbreviation' : 'id';
-  $sql = "SELECT $ret FROM civicrm_state_province WHERE $input_type = %1";
-  $vars = array(1 => array($input, $ret == 'id' ? 'String' : 'Integer'));
+  $params = [];
+  if ($ret == 'id') {
+    $params['abbreviation'] = $input;
+  }
+  else {
+    $params['id'] = $input;
+  }
   if ($ret === 'id') {
     if (!$country_id || $country_id === 'default') {
-      $config = CRM_Core_Config::singleton();
-      $country_id = (int) $config->defaultContactCountry;
+      $settings = wf_civicrm_api('Setting', 'get', [
+        'sequential' => 1,
+        'return' => 'defaultContactCountry',
+      ]);
```

Comment:
So in the 7.x-5.x branch we have a function `wf_crm_get_civi_setting()` which was introduced in 3da7c660406217a23d759dcf2772b7221aafec11. Looks like that hasn't yet been ported to the 8.x branch but it would be nice. Not a blocker to this PR tho.

Reply:
I've submitted a PR to port those changes so actually this PR will need rebasing once #239 is merged.